src/ensembl/src/shared/components/help-popup/HelpPopupButton.tsx

```diff
@@ -28,7 +29,16 @@ import { SlugReference } from './types';
 
 import styles from './HelpPopupButton.scss';
 
-type Props = SlugReference;
+type ClassNames = Partial<{
+  label: string;
+  button: string;
+}>;
+
+type Props = {
+  classNames?: ClassNames;
+  text: string;
+  slug: SlugReference;
+};
```

Comment:
Let's have a think about the props.

The component seems to exist in two states:
1) Only the button itself is clickable
2) Both the button and the label next to it are clickable

When the label text is grey, like here:

<img src="https://user-images.githubusercontent.com/6834224/127164708-0ee2bced-4167-480b-8616-88c0846b5d66.png" width="100"/>

I wouldn't expect it to be clickable, but in this PR, it is.

So I would expect the Props to have a shape that is approximately like this:

```
type Props = {
  slug: string;
  label: string;
  isLabelClickable: boolean;
}
```

If the label is clickable, it will be styled as blue and have a pointer cursor. If it isn't it will be styled as grey and not change the cursor to a pointer.

Reply:
The grey text in other pages is still clickable though, isn't it? It's just the homepage's help text that is in blue colour as far as I know.

Reply:
It is on internal, that's true; but that would violate our general style guide, where grey isn't clickable. It's possible that Andrea had a different opinion on this particular instance of grey, and as a result I made it clickable. Probably should ask her again to be sure.

Reply:
Changed the props to:

```
type Props = {
  slug: string;
  labelClass?: string;
  text: string;
}
```

Reply:
✅

Reply:
Can I also ask to change the name of the `text` property to `label`, to make it obvious that it refers to the same thing as the `labelClass`?

Reply:
That's done now ✅